src/lite-psm-halt/SingleLitePsmHaltSpell.t.integration.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.16;
+
+import {stdStorage, StdStorage} from "forge-std/Test.sol";
+import {DssTest, DssInstance, MCD, GodMode} from "dss-test/DssTest.sol";
+import {DssEmergencySpellLike} from "../DssEmergencySpell.sol";
+import {SingleLitePsmHaltSpellFactory, LitePsmLike, Flow} from "./SingleLitePsmHaltSpell.sol";
+
+contract SingleLitePsmHaltSpellTest is DssTest {
+    using stdStorage for StdStorage;
+
+    address constant CHAINLOG = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
+    DssInstance dss;
+    address chief;
+    address litePsmMom;
+    LitePsmLike psm;
+    SingleLitePsmHaltSpellFactory factory;
+
```

Comment:
Nit: extra blank line. Make sure to run `forge fmt` in the file to prevent that.
```suggestion
```

Reply:
Done in [chore: forge fmt](https://github.com/makerdao/dss-emergency-spells/pull/2/commits/5279310bf5ef874950766065ee0fc845b926afb3)